src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/AutomaticUserConfirmationPolicyRequirement.cs

```diff
@@ -19,10 +19,11 @@ namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements
 /// <param name="policyDetails">Collection of policy details that apply to this user id</param>
 public class AutomaticUserConfirmationPolicyRequirement(IEnumerable<PolicyDetails> policyDetails) : IPolicyRequirement
 {
-    public bool IsEnabled(Guid organizationId) => policyDetails.Any(p => p.OrganizationId == organizationId);
+    public bool CanBeGrantedEmergencyAccess() => policyDetails.Any();
+
+    public bool UserBelongsToOrganizationWithAutomaticUserConfirmationEnabled() => policyDetails.Any();
```

Comment:
`IPolicyRequirement`s work best when they express what the policy requires in business logic terms. This isn't always possible, but here this could be (for example) `CanJoinProvider()`. Then that's the interface that the `ProviderService` calls.